contrib/kl_layer/README.md

```diff
@@ -0,0 +1 @@
+This folder contains a self-contained example
```

Comment:
Can you add a bit more of a description--a sentence or two about the KL loss--and what the two files are?

Reply:
yeah,sorry, I was still working on it. Will finish asap